action/execution.go

```diff
@@ -64,6 +64,30 @@ func NewExecution(
 	}, nil
 }
 
+// NewExecution returns a Execution instance
+func NewExecution(
+	contractAddress string,
+	nonce uint64,
+	amount *big.Int,
+	gasLimit uint64,
+	gasPrice *big.Int,
+	data []byte,
+	list types.AccessList,
+) (*Execution, error) {
+	return &Execution{
```

Comment:
https://github.com/iotexproject/iotex-core/pull/3405/files/52b5039dc0fad1e75aad6a6801ed5cfa95d7a05d..5d1ec60d88e831002efcb4a761416e327b4e87ea#r887155645

Of cause, I prefer your current implementation.